src/validator/profile.ts

```diff
@@ -1,9 +1,12 @@
 import { Request, Response, NextFunction } from 'express';
 export const updateProfileValidator = (req: Request, res: Response, next: NextFunction) => {
+  // TODO is only checking for length but not what is allowed etc.
   req.check("description", "description should be less than 256 character").isLength({
     min: 0,
     max: 256
   });
+  // allow alphabet, number, -, _, . only 
+  req.check("description", "description contains invalid character").matches(/^[a-z\d\-_.!@#\s]+$/i)
```

Comment:
Whitelist alphanumeric + number + whitespace + -_.!@# for now, how about this?

Reply:
also i fixed user param request error handling(which i forgot again :/ ) and testing accordingly fyi